runtime/Python2/src/antlr4/InputStream.py

```diff
@@ -21,7 +21,7 @@ def __init__(self, data):
 
     def _loadString(self):
         self._index = 0
-        self.data = [ord(c) for c in self.strdata]
+        self.data = list(from_unicode(self.strdata))
```

Comment:
Yes I had only looked at _from_unicode, not great that these 2 don't return the same type, maybe you want to make _from_unicode return a generator too, then in the future we can benefit from lazy conversion if clients of self.data support it